src/OrchardCore.Modules/OrchardCore.AuditTrail/Navigation/AuditTrailAdminMenu.cs

```diff
@@ -23,11 +23,12 @@ public Task BuildNavigationAsync(string name, NavigationBuilder builder)
             }
 
             builder
-                .Add(S["Audit Trail"], NavigationConstants.AdminMenuAuditTrailPosition, configuration => configuration
+                .Add(S["Audit trail"], NavigationConstants.AdminMenuAuditTrailPosition, audit => audit
```

Comment:
Out of scope, and these changes are actually breaking the translations

Reply:
@sebastienros what is out of scope changing `S["Audit Trail"]` to `S["Audit trail"]` or standardizing how the menu items appear on the menu?

Reply:
Changing the menu names break translations. So, he suggests making a second PR for that matter since it's a different scope than the actual PR.

Reply:
I see. I has been resolved. I think we should create another issue for creating consistent formatting including defining the standards.